clang/include/clang/Driver/Options.td

```diff
@@ -240,7 +240,8 @@ def m_riscv_Features_Group : OptionGroup<"<riscv features group>">,
 def m_ve_Features_Group : OptionGroup<"<ve features group>">,
                           Group<m_Group>, DocName<"VE">;
 def m_loongarch_Features_Group : OptionGroup<"<loongarch features group>">,
-                                 Group<m_Group>, DocName<"LoongArch">;
+                                 Group<m_Group>, DocName<"LoongArch">,
+                                 Visibility<[ClangOption, CLOption, FlangOption]>;
```

Comment:
Seems `CLOption` is not related to this change.

Reply:
```
def m_Group : OptionGroup<"<m group>">, Group<CompileOnly_Group>,
              DocName<"Target-dependent compilation options">,
              Visibility<[ClangOption, CLOption]>;
```

I wondor if it inherits from `m_Group` when `Visibility` is not specified. If so, it originally contained `CLOption`.

Reply:
Yes. It should be.